Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py

```diff
@@ -502,32 +459,12 @@ def test_quarantine_file_command(mocker):  # _with_connection
     raw_response = json.loads(load_mock_response('get_remediation_action.json'))
     mocker.patch("Cybereason.get_remediation_action", return_value=raw_response)
     raw_response = json.loads(load_mock_response('get_remediation_action_status.json'))
-    mocker.patch("Cybereason.get_remediation_action_status", return_value=raw_response)    
+    mocker.patch("Cybereason.get_remediation_action_status", return_value=raw_response)
     with pytest.raises(Exception):
-        command_output = quarantine_file_command(client, args)
+        quarantine_file_command(client, args)
 
```

Comment:
I suggest checking the content of the exception to make sure the function failed for the right reasons.

Reply:
Done